Sources/Testing/Parameterization/Test.Case.ID.Selection.swift

```diff
@@ -0,0 +1,61 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension Test.Case.ID {
+  /// A selection of ``Test/Case/ID`` instances.
+  struct Selection: Sendable {
```

Comment:
Should we make `Selection` generic over some `T: Sendable & Identifiable & ?`?

Reply:
Are you asking whether we should make `Test.ID.Selection` generic, or `Test.Case.ID.Selection`? Note that `Test.ID.Selection` has different semantics: its "contains" operation considers ancestors and descendants

Reply:
I'm suggesting they're all the same thing.